Frontend/src/components/JobListing.js

```diff
@@ -32,15 +32,7 @@ const JobListing = ({ listingObj, setPopupOpen }) => {
   const date = parseDate(listingObj.createdAt);
 
   return (
-      <li 
-        className={hover ? 'job-listing-hover' : 'job-listing'} 
-        onMouseEnter={() => setHover(true)}
-        onMouseLeave={() => setHover(false)}
-        onClick={() => {
-          history.push(`${url}job/${listingObj.id}`);
-          setPopupOpen(listingObj);
```

Comment:
Looks like the job-popup has stopped working because of deleting the `onClick` prop, just wanted to confirm if it was intentional or not. Looks great otherwise!

Reply:
Oh, that wasn't me. Thanks!